scripts/ci/build-and-test.sh

```diff
@@ -10,20 +10,29 @@ cd $(dirname $0)/../..
 source scripts/ci/sources/mode.sh
 source scripts/ci/sources/tunnel.sh
 
+# Check if tests can be skipped
+skip_tests=true
+for filename in $(git diff --name-only $TRAVIS_BRANCH...HEAD); do
```

Comment:
But If I recall correctly, for Push builds the `$TRAVIS_BRANCH` will be `master` and the merged commit will be `HEAD` / `master` as well.

This would mean that the `diff` will be always empty for push builds.